src/test/java/org/apache/maven/plugins/javadoc/JavadocJarTest.java

```diff
@@ -154,7 +154,7 @@ public void testInvalidDestdir()
         //check if the javadoc jar file was generated
         File generatedFile = new File( getBasedir(),
                                        "target/test/unit/javadocjar-invalid-destdir/target/javadocjar-invalid-destdir-javadoc.jar" );
-        assertTrue( !FileUtils.fileExists( generatedFile.getAbsolutePath() ) );
+        assertFalse(FileUtils.fileExists(generatedFile.getAbsolutePath()));

```

Comment:
Maven style uses spaces before ) and after ( unless there are no arguments

Reply:
@elharo done.
btw, is there a code formatter / style checker for this code for this format we used here?

Reply:
please check here: https://maven.apache.org/developers/conventions/code.html

Reply:
> please check here: https://maven.apache.org/developers/conventions/code.html

@slachiewicz Thanks.
btw is there anybody for more reviewings to this pr?